http/src/main/resources/jupyter/init-actions.sh

```diff
@@ -368,6 +369,15 @@ END
         fi
       fi
 
+      # If a Jupyter start user script was specified, copy it into the jupyter docker container for consumption by startup.sh.
+      if [ ! -z ${JUPYTER_START_USER_SCRIPT_URI} ] ; then
```

Comment:
I don't see where `$JUPYTER_START_USER_SCRIPT_URI` is being set… it may need to be exported in the top of `init-actions.sh` and templated in by Leo

Reply:
Based on further reading it appears it is templated by Leo, so just needs to be exported above

Reply:
Oops thanks, forgot to add that. Done